iiif_prezi3/helpers/set_hwd.py

```diff
@@ -0,0 +1,29 @@
+from ..loader import monkeypatch_schema
+from ..skeleton import Canvas, ResourceItem
+
+
+class SetHwd:
+    def set_hwd(self, height, width, duration):
```

Comment:
suggest setting default values of `None` so can be called with keyword args

```suggestion
    def set_hwd(self, height=None, width=None, duration=None):
```

Reply:
+1 to Rebecca's suggestion

Reply:
Thanks Rebecca! Thats a good idea!